components/api/src/main/java/com/hotels/styx/api/cookies/ResponseCookie.java

```diff
@@ -224,20 +223,21 @@ public boolean equals(Object o) {
 
     @Override
     public int hashCode() {
-        return hash(name, value, domain, maxAge, path, httpOnly, secure);
+        return hashCode;
     }
 
     @Override
     public String toString() {
-        return toStringHelper(this)
-                .add("name", name)
-                .add("value", value)
-                .add("domain", domain)
-                .add("maxAge", maxAge)
-                .add("path", path)
-                .add("httpOnly", httpOnly)
-                .add("secure", secure)
-                .toString();
+        final StringBuilder sb = new StringBuilder("ResponseCookie{");
```

Comment:
This could be simplified to:
```
return new StringBuilder("ResponseCookie{")
                .append("name='").append(name).append('\'')
                .append(", value='").append(value).append('\'')
                .append(", domain='").append(domain).append('\'')
                .append(", maxAge=").append(maxAge)
                .append(", path='").append(path).append('\'')
                .append(", httpOnly=").append(httpOnly)
                .append(", secure=").append(secure)
                .append('}').toString();
```
or simply
```
return "ResponseCookie{" +
                "name='" + name + '\'' +
                ", value='" + value + '\'' +
                ", domain='" + domain + '\'' +
                ", maxAge=" + maxAge +
                ", path='" + path + '\'' +
                ", httpOnly=" + httpOnly +
                ", secure=" + secure +
                '}';
```

Reply:
Addressed & pushed changes.